scripts/generate/blockImporterTables.sql

```diff
@@ -10,19 +10,25 @@ CREATE TABLE utxos  ( utxo_id   text      PRIMARY KEY
           					, amount 	  bigint
                     );
 
+-- FIXME: Not used, delete?
 CREATE TABLE bestBlock ( best_block_num bigint);
 
-CREATE TABLE txs 	( hash		  hash PRIMARY KEY
-        					, block_num bigint NULL
-        					, time      timestamp with time zone NULL
+CREATE TABLE txs 	( hash		          hash      PRIMARY KEY
```

Comment:
I think it's better to remove this custom types and use `text` instead. The same applies to all columns